test/intl402/NumberFormat/constructor-options-toobject.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2018 Ujjwal Sharma. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-initializenumberformat
+description: >
+  Tests that Intl.NumberFormat contructor converts the options argument
+  to an object using `ToObject` (7.1.13).
+info: |
+  11.1.2 InitializeNumberFormat
+
+  3.a. Let options be ? ToObject(options).
+---*/
+
+const toObjectResults = [
+  [true, new Boolean(true)],
+  [42, new Number(42)],
+  ['foo', new String('foo')]
+];
```

Comment:
What about Symbol? Object?

Reply:
Objects are returned as-is by `ToObject`, so I didn't think that'd create much of a difference. Symbols don't have a primitive (right?) so it'll just be making a new Symbol like the one that was passed (plus also no `new`), so I thought that won't make a huge difference as well.

If you had suggestions about how to test for them in a useful way, I'd love to add things to this but I thought it would be pointless to compare results between (a) `{}` and `{}` (b) `Symbol('foo')` and `Symbol(Symbol('foo'))`